spec/helpers/admin/users_helper_spec.rb

```diff
@@ -1,6 +1,33 @@
 require "rails_helper"
 
 describe Admin::UsersHelper do
+  describe "#role_options" do
+    let(:user) { create(:user) }
+
+    it "returns base roles" do
+      user.add_role(:admin)
+      roles = helper.role_options(user)
+      expect(roles).to have_key("Base Roles")
+      expect(roles["Base Roles"]).to eq Constants::Role::BASE_ROLES
+    end
+
+    it "returns special roles" do
+      user.add_role(:super_admin)
+      roles = helper.role_options(user)
+      expect(roles).to have_key("Special Roles")
+      expect(roles["Special Roles"]).to eq Constants::Role::SPECIAL_ROLES
+    end
+
+    it "adds moderator role when feature flag enabled" do
+      user.add_role(:super_admin)
+      FeatureFlag.enable(:moderator_role)
```

Comment:
(nitpick): @dscottS3 do you think we'd be able to instead stub the Feature Flag so that it remains scoped to this test?

```
      allow(FeatureFlag).to receive(:enabled?).with(: moderator_role).and_return(true)
```

I think that we're planning on removing the feature flag once the functionality is built so not a blocker.